lib/rubotnik.rb

```diff
@@ -23,7 +23,8 @@ def self.subscribe(token)
   end
 
   def self.route(event, &block)
-    Bot.on(event, &block) unless [:message, :postback].include?(event)
+    return Bot.on(event, &block) unless [:message, :postback].include?(event)
```

Comment:
This was actually a bug - e.g. `:deliver` events would be handled by the next line anyways. `return` fixes it.

Reply:
I think the `return` here may break stuff, there should be exactly one instance of `Bot` (facebook-messenger) so that everything runs in the same process for a same user, at least that was my understanding at the time of writing it first. I'd rather make a conditional there, as I recognize it's currently a bug. Like so: 

```
def self.route(event, &block)
    if [:message, :postback].include?(event)
      # handle with Rubotnik
    else
      # handle without Rubotnik
    end
  end
```

Reply:
Good idea 👍